client/webserver/site/src/html/bodybuilder.tmpl

```diff
@@ -8,7 +8,7 @@
   <link rel="icon" href="/img/favicon.png?v=AK4XS4">
   <meta name="description" content="Decred DEX Client Web Portal">
   <title>{{.Title}}</title>
-  <link href="/css/style.css?v=LESXCTh9d" rel="stylesheet">
+  <link href="/css/style.css?v=GHSXZ15s" rel="stylesheet">
```

Comment:
There style bundle is unchanged.

Reply:
I think this must have been during my last rebase.